hyperspy/_signals/lazy.py

```diff
@@ -23,7 +23,8 @@
 
 import numpy as np
 import dask.array as da
-import dask
+import dask.threaded as dt
+from dask import __version__ as daskversion
```

Comment:
LGTM recommends not to combine `import` and `import from` for the same module for maintainability: https://lgtm.com/rules/1818040193/

Maybe it would be better to do it the other way round and remove the `import from` instead of converting the other imports to additional `import from` statements. If you agree, I would change it that way round.